src/components/TransactionsTable.vue

```diff
@@ -49,6 +68,10 @@ defineProps({
     required: true,
   },
 })
+
+function getHintKey(type) {
+  return type.charAt(0).toLowerCase() + type.slice(1)
```

Comment:
I propose to throw an error when a new type of transaction is added, that doesn't have a hint. This way we will know that something is missing. In such case, we should most likely also not display the hint-tooltip at all until an appropriate caption for that transaction type is implemented.

Reply:
Maybe I am wrong, but I am not aware of any plans of adding new tx types in the future.

Reply:
I'm not aware of such plans either but it doesn't change the fact that we should try to make our code future-proof, especially in low-cost cases like this one :) 

I'll leave @michele-franchi to be the judge here and approve in advance

Reply:
If it works then let's leave it like it is. 😄